x-pack/legacy/plugins/maps/public/layers/layer.js

```diff
@@ -333,10 +342,6 @@ export class AbstractLayer {
     return [];
   }
 
-  async getFields() {
```

Comment:
Moved to VectorLayer. The concept of fields does not make sense for tile layers and so on. It really is a VectorLayer concept

Reply:
thanks, +1